merkle/merkle.go

```diff
@@ -0,0 +1,45 @@
+package merkle
+
+import (
+	"context"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/avalanchego/database/memdb"
+	"github.com/ava-labs/avalanchego/trace"
+	"github.com/ava-labs/avalanchego/x/merkledb"
+	"github.com/ava-labs/hypersdk/utils"
+	"github.com/ava-labs/avalanchego/ids"
+)
+
+// Generate merkle root for a set of items
+func GenerateMerkleRoot(ctx context.Context, config merkledb.Config, tracer trace.Tracer, merkleItems [][]byte, consumeBytes bool) (ids.ID, merkledb.MerkleDB, error) {
```

Comment:
tracer is provided in `merkledb.Config`, so this arg can be removed.

Reply:
removed!